src/rules/no-unused-modules.js

```diff
@@ -201,6 +203,42 @@ const newNamespaceImportExists = specifiers =>
 const newDefaultImportExists = specifiers =>
   specifiers.some(({ type }) => type === IMPORT_DEFAULT_SPECIFIER)
 
+const fileIsInPkg = file => {
+  const { path, pkg } = readPkgUp.sync({cwd: file, normalize: false})
+  const basePath = dirname(path)
+
+  if (pkg.private === true) {
+    return false
+  }
+
+  if (pkg.bin && typeof pkg.bin === 'string') {
+    if (join(basePath, pkg.bin) === file) {
+      return true
+    }
+  }
+
+  if (pkg.bin && typeof pkg.bin === 'object') {
+    const binFiles = Object.values(pkg.bin).map(value => join(basePath, value))
+    if (binFiles.includes(file)) {
+      return true
+    }
+  }
+
+  if (pkg.browser) {
```

Comment:
The browser field can also be an object.

Reply:
Thanks for this info. This wasn't mentioned in the [npm documentation](https://docs.npmjs.com/files/package.json). I will quickly add this part.

Reply:
https://github.com/defunctzombie/package-browser-field-spec